app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsFragment.java

```diff
@@ -185,8 +185,8 @@ protected void groupByDateSessions(List<Session> sessions) {
         }
     }
 
-    private void addFragment(String title, List<Session> sessions) {
-        SessionsTabFragment fragment = SessionsTabFragment.newInstance(sessions);
+    private void addFragment(String title, List<Session> sessions, @SessionsTabFragment.RENDER_MODE int mode) {
+        SessionsTabFragment fragment = SessionsTabFragment.newInstance(sessions, mode);
```

Comment:
One suggestion
I guess it is better to change this method to `protected`, and override each chilren class. Because RENDER_MODE is only used here. What do you think?